src/components/editor/EditorAndList.tsx

```diff
@@ -0,0 +1,41 @@
+import { Grid } from '@mui/material';
+import EditorFileSelector from 'components/editor/FileSelector';
+import MonacoEditor, {
+    Props as MonacoEditorProps,
+} from 'components/editor/MonacoEditor';
+
+export interface Props extends MonacoEditorProps {
+    height?: number;
+}
+
+const DEFAULT_HEIGHT = 350;
+
+function EditorAndList(props: Props) {
+    const { height } = props;
+    const heightVal = height ?? DEFAULT_HEIGHT;
+
+    return (
```

Comment:
Most of this is the same as before. However, we are now using a grid to lay stuff out. This is much nicer than before and helps make Monaco editor behave.

Reply:
I think we still need to do some work on the "responsiveness" of the application.